includes/Elements/Flip_Box.php

```diff
@@ -1588,6 +1588,7 @@ protected function render()
 		                    if ( ! is_array( $settings['eael_flipbox_front_templates'] ) ) {
 			                    $settings['eael_flipbox_front_templates'] = apply_filters( 'wpml_object_id', $settings['eael_flipbox_front_templates'], 'wp_template', true );
 		                    }
+                            // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Looks like alignment issue with the next (below) line

Reply:
In local editor it looks fine. 
<img width="1322" alt="image" src="https://github.com/user-attachments/assets/82629194-52c6-47ad-b1f0-a97909a9e9f8">